src/Orleans.Streaming/Providers/SiloStreamProviderRuntime.cs

```diff
@@ -6,9 +6,13 @@
 using Microsoft.Extensions.Logging;
 using Orleans.Configuration;
 using Orleans.Streams.Filtering;
+using Orleans.Internal;
 
 namespace Orleans.Runtime.Providers
 {
+    public interface IMessageDeliveryBackoffProvider : IBackoffProvider { }
```

Comment:
Could you move these declaration in another file (one file for both of them is fine to me), and add xml-doc on them ?